packages/mermaid/src/config.type.ts

```diff
@@ -411,6 +412,26 @@ export interface FlowchartDiagramConfig extends BaseDiagramConfig {
   wrappingWidth?: number;
 }
 
+export enum SankeyLinkColor {
+  source = 'source',
+  target = 'target',
+  gradient = 'gradient',
+}
+
+export enum SankeyNodeAlignment {
+  left = 'left',
+  right = 'right',
+  center = 'center',
+  justify = 'justify',
+}
```

Comment:
Hi @nirname, sorry I didn't get a chance to look at this PR before it was merged. The new Sankey diagrams look really good!

TypeScript enums have a bunch of issues, so I've made a PR changing these lines of code to remove the `enum`, see https://github.com/mermaid-js/mermaid/pull/4580.

Please feel free to review this PR with your feedback!